### PR TITLE
add print statements to run script and to my 2 scripts

### PR DESCRIPTION
**Pull request by bolliger32**
provide some print statements to keep people aware of whats goin on

This is just print statements and some auto-formatting, so shouldn't change behavior at all

### CODE REVIEW

File: codes/data/multi_country/get_adm_info.py [hunk 1]
```diff
@@ -215,6 +224,7 @@ def main():
         }
     }
     for iso3 in isos:
+        print(f"Downloading and processing {iso3} geographical and population data...")
```
Comment (kendonB) [inline codes/data/multi_country/get_adm_info.py]: check the f prefix

File: codes/data/multi_country/get_adm_info.py [hunk 2]
```diff
@@ -408,23 +411,26 @@ def main():
...
 
-
     # ##### Merge back into global adm datasets
 
     adm2_gdf = adm2_gdf.fillna(us_pops)
     st_pops = (
-        adm2_gdf.loc[:, "population"].groupby(["adm0_name", "adm1_name"]).sum(min_count=1)
+        adm2_gdf.loc[:, "population"]
+        .groupby(["adm0_name", "adm1_name"])
+        .sum(min_count=1)
```
Comment (kendonB) [inline codes/data/multi_country/get_adm_info.py]: does this work?

Comment (kendonB) [reply codes/data/multi_country/get_adm_info.py]: With no evidence nor knowledge my heart feels like the dots should go the line above

Comment (bolliger32) [reply codes/data/multi_country/get_adm_info.py]: it works! FWIW this was using `black` an autoformatter. I've never seen it actually break things b/c it's pretty conservative but it makes your code look nice and pretty. You just call `black path_to_file`

File: codes/data/multi_country/get_adm_info.py (hunk 1 above)
Comment (bolliger32) [reply codes/data/multi_country/get_adm_info.py]: good call! will fix

Comment (bolliger32) [reply codes/data/multi_country/get_adm_info.py]: oh whoops nevermind thats right. it's an fstring in python...which is the same as `"blahblah{a}".format(a=someothervar)`

**Pull request merged**